ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationHomeOwnersIF.ts

```diff
@@ -0,0 +1,14 @@
+import { AddressIF } from "@/composables/address/interfaces"
+
+export interface MhrRegistrationHomeOwnersIF {
+  id?: string, // optional property used for editing a home owner
+  individualName: {
+    first: string,
+    middle: string,
+    last: string
+  },
+  phoneNumber: number,
+  phoneExtension: number
+  address: AddressIF
+  // type: string
+}
```

Comment:
Is the plan to include Organizations here, future*?

Reply:
Yes, based on API (once finalized) this IF will be updated.